### ISSUE TITLE
Train/Test split when training linear classifier (ImageNet)

### ISSUE THREAD

**Issue by jnyjxn** [HobbitLong/CMC#53]
Hello,

Thank you for this great work and well-written paper.

When training the linear classifier (e.g. Alexnet with ImageNet), how do you perform the train/test split? 1. Do you use all of the classes in your test set?
2. How do you choose views for each sample?

Many thanks,
Jonny

**Comment by HobbitLong** [comment]
Hi Jonny,

1. ImageNet dataset itself has a "train" set and a "val" set, which will be used as train and test sets, respectively.
2. The same as the CMC pre-training stage. If you train CMC with Y/DbDr split, then you use Y/DbDr split in linear evaluation stage as well.

**Comment by jnyjxn** [comment]
Thanks @HobbitLong - unfortunately from what I can tell ImageNet is defunct (I've had no response for > 3 months) so I can't access the dataset.

Could you briefly describe the approach they've used to do the split? Specifically, do they split according to e.g. 80 different chairs in train, 20 different chairs in val, or is it the same 100 chairs in train and val but with different perspectives?

Thank you!

**Comment by HobbitLong** [comment]
I believe it's the former way.
